### DISCUSSION TITLE
SH1mmer Issues

### DISCUSSION THREAD

**Discussion by Merlinxx10** [3kh0/ext-remover#543] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I don't know if this is a helpable issue but after setting my Chromebook into recovery mode and using the USB I get to a part that requires a QR code and another text above-saying something about developer mode set to off by device policy or something like that and how I need to disable the writing that's making it say that or something is there a way to fix this so I can access the SH1mmer GUI or what because I Don't know what to do now.

**Comment by obkduishinijf** [comment]
you.. don't go into dev mode path.... you go into recovery mode.

**Comment by amelia-bartlett** [comment]
> I don't know if this is a helpable issue but after setting my Chromebook into recovery mode and using the USB I get to a part that requires a QR code and another text above-saying something about developer mode set to off by device policy or something like that and how I need to disable the writing that's making it say that or something is there a way to fix this so I can access the SH1mmer GUI or what because I Don't know what to do now.

you mean this? https://github.com/MercuryWorkshop/sh1mmer/issues/4

**Comment by Br1tBreaker** [comment, marked as answer]
bro
It sounds like your Chromebook is stuck in a recovery mode loop and is asking for a QR code to proceed. This issue may be caused by a variety of factors, such as corrupt system files or a failed update.

Here are some steps you can try to resolve this issue:

    Try a hard reset: Hold down the power button on your Chromebook for at least 10 seconds to force a shutdown, then turn it back on and see if the issue persists.

    Try a different recovery image: Make sure that you have the correct recovery image for your Chromebook model. If the recovery image you are using is not working, try downloading a different one.

    Disable device policy: If your Chromebook is managed by an organization, it may have device policies that prevent you from accessing certain features. In this case, you can try disabling the policy by following these steps:

    Press Ctrl+Alt+T to open the Chrome OS terminal.
    Type "shell" and press Enter.
    Type "sudo su" and press Enter.
    Type "crossystem dev_boot_signed_only=0" and press Enter.
    Type "exit" and press Enter.
